package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "gutenberg",
-	"version": "4.7.1",
+	"version": "4.8.0",
```

Comment:
Should we update the version (and publish) other packages changed since the last publish process?

Reply:
Yes, we don't publish packages with plugin releases anymore, packages updates will be done from the `g-minor` branch once we cherrypick all the merged 5.0.3 fixes there.